CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.1.1
+
+### bug fixes
```

Comment:
Actually this should probably be under the `Object Server API Changes` header just like for `2.1.0`